yarn.lock

```diff
@@ -8360,7 +8360,7 @@ jake@^10.8.5:
     filelist "^1.0.1"
     minimatch "^3.0.4"
 
-jasmine-core@^3.6.0:
+jasmine-core@3.10.1, jasmine-core@^3.6.0:
```

Comment:
Unrelated, fixed in https://github.com/salesforce/lwc/pull/3349

Reply:
Rebased and this went away.